extensions/amp-consent/0.1/test-e2e/test-cmp-fullscreen.js

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
I wonder if you want to add additional UI test here. For example the lightbox mode. If so I might rename the file. 
But that can also be a visual diff test. So your call

Reply:
I like the idea of doing more e2e tests, so I'll change the file name.